composer.json

```diff
@@ -15,7 +15,8 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0",
+        "php": ">=5.4",
+        "ext-intl": "*",
```

Comment:
Yes, as long as it would break backward compatibility. It can be only suggested to use new keymaker but it is not required.